phylo_workflow/find_clonal.py

```diff
@@ -0,0 +1,64 @@
+import time
+import tskit
+import numpy as np
+import msprime
+import collections
+import sys
+
+# All this code assumes a single tree
+
+def num_mutations_above_node(ts):
+    out = np.zeros(dtype=np.int32, shape=ts.num_nodes)
```

Comment:
I think this function can just be ``np.bincount(ts.tables.mutation.node)``

Reply:
Nice! Hadn't come across that numpy function before.